src/components/AddItem.jsx

```diff
@@ -1,7 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { collection, addDoc } from 'firebase/firestore';
+import { db } from '../lib/firebase';
+
+const radioButtonOptions = [
+  {
+    option: 'soon',
+    value: 7,
+    defaultChecked: true,
+  },
+  {
+    option: 'kind of soon',
+    value: 14,
+    defaultChecked: false,
+  },
+  {
+    option: 'not soon',
+    value: 30,
+    defaultChecked: false,
+  },
+];
 
 function AddItem() {
-  return <div>Add Item</div>;
+  const [itemName, setItemName] = useState('');
+  const [frequency, setFrequency] = useState(7);
+  const [notification, setNotification] = useState('');
+
+  //retrive the token from localStorage
+  const token = localStorage.getItem('list-token');
+
+  //handle submit for the form
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    //create new doc reference for text input and save to firestore database
+    //use a token to save new items into specific collection (shopping list)
+    try {
+      const docRef = await addDoc(collection(db, token), {
```

Comment:
I'm having trouble adding items, and I think it might be because of this addDoc method. When I click the Add Item button, I get the following error in my console - ```Error adding document: FirebaseError: Function collection() cannot be called with an empty path.```. I think the argument after db needs to be the name of the collection (as a string) in Firestore. In this case, I think we were using 'items' before.

Reply:
Can it be because you don't have token saved into your localStorage? 
For now we are reading a token from localstorage and then save items into document, name of this doc = token value. It is how we understood our shopping list would be organized. You create an unic token and we create a shopping list under this token and save items inside this list/collection.

Reply:
Also, I was thinking to avoid situation when user doesn't have token saved in localStorage to redirect user back to main page where user will be asked to create new list.  If you guys think it makes sense we can add it.

Reply:
I don't have a token in localStorage. I'm not seeing how I would get a token in this branch unless I set it manually? Once we merge, our branch has the functionality to check for a token and create a new list if there is not one present, so I don't think you need to do that here.

Reply:
You can set it manually for now to test our code